src/CssInliner.php

```diff
@@ -150,9 +133,15 @@ class CssInliner extends AbstractHtmlProcessor
 
     /**
      * array of data describing CSS rules which apply to the document but cannot be inlined, in the format returned by
-     * `parseCssRules`
+     * {@see collateCssRules}
```

Comment:
Why the curly braces around the `@see` annotation?

Reply:
It's meant to be inline as part of the description, to appear as a link, which is why there's no blank line before it.  It just so happens that the 120 character line length pushes it onto the next line.